### PR TITLE
Feature(Share-CRR): Cross region replication support for file share

### PR DESCRIPTION
**Pull request by deepaksibm**

### CODE REVIEW

**Comment by deepaksibm** [comment]
```
$ tfpt TestAccIbmIsShareCrossRegionReplication
=== RUN   TestAccIbmIsShareCrossRegionReplication
--- PASS: TestAccIbmIsShareCrossRegionReplication (89.05s)
PASS
ok  	github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc	90.196s
```

File: website/docs/d/is_share.html.markdown [hunk 1]
```diff
@@ -51,6 +51,11 @@ The following attributes are exported:
 - `encryption_key` - The CRN of the key used to encrypt this file share. Nested `encryption_key` blocks have the following structure:
```
Comment (SunithaGudisagarIBM1) [inline website/docs/d/is_share.html.markdown:51]: Nested `encryption_key` blocks have the following structure: remove here also

File: ibm/acctest/acctest.go [hunk 2]
```diff
@@ -815,6 +817,18 @@ func init() {
 		fmt.Println("[INFO] Set the environment variable IS_SHARE_PROFILE for testing ibm_is_instance resource else it is set to default value 'tier-3iops'")
 	}
 
+	SourceShareCRN = os.Getenv("IS_SOURCE_SHARE_CRN")
+	if SourceShareCRN == "" {
+		SourceShareCRN = "crn:v1:staging:public:is:us-east-1:a/efe5afc483594adaa8325e2b4d1290df::share:r142-a106f162-86e4-4d7f-be75-193cc55a93e9" // for next gen infrastructure
```
Comment (SunithaGudisagarIBM1) [inline ibm/acctest/acctest.go:822]: SourceShareCRN is from us-east-1 and ShareEncryptionKey is us-south, does this combination work?

**Review by SunithaGudisagarIBM1 — APPROVED** [review]
Please find my small comments here.
Post the incorporation PR is ready with code review

**Comment by SunithaGudisagarIBM1** [comment]
May be we can try to change/update the encryption_key, source_share_crn and initial_owner and attach the results in jira

**Comment by deepaksibm** [comment]
upstream merged

**Pull request closed without merging**